portfolio/src/main/webapp/script.js

```diff
@@ -64,3 +64,207 @@ function clearComments() {
+            stylers: [{color: '#9ca5b3'}]
+          },
+          {
+            featureType: 'road.highway',
+            elementType: 'geometry',
+            stylers: [{color: '#616163'}]
+          },
+          {
+            featureType: 'road.highway',
+            elementType: 'geometry.stroke',
+            stylers: [{color: '#1f2835'}]
+          },
+          {
+            featureType: 'road.highway',
+            elementType: 'labels.text.fill',
+            stylers: [{color: '#f3d19c'}]
+          },
+          {
+            featureType: 'transit',
+            elementType: 'geometry',
+            stylers: [{color: '#2f3948'}]
+          },
+          {
+            featureType: 'transit.station',
+            elementType: 'labels.text.fill',
+            stylers: [{color: '#d59563'}]
+          },
+          {
+            featureType: 'water',
+            elementType: 'geometry',
+            stylers: [{color: '#17263c'}]
+          },
+          {
+            featureType: 'water',
+            elementType: 'labels.text.fill',
+            stylers: [{color: '#515c6d'}]
+          },
+          {
+            featureType: 'water',
+            elementType: 'labels.text.stroke',
+            stylers: [{color: '#17263c'}]
+          }
+        ]
+      });
+
+  // Store urls in String variables for readability and easiness to follow
+  const shabuShabuWikiUrl = 'https://en.wikipedia.org/wiki/Shabu-shabu';
+  const koreanBarbecueWikiUrl = 'https://en.wikipedia.org/wiki/Korean_barbecue';
+  const hotPotWikiUrl = 'https://en.wikipedia.org/wiki/Hot_pot';
+  const boilingPointUrl = 'https://www.bpgroupusa.com/';
+  const inNOutUrl = 'https://www.in-n-out.com/';
+  const clydesHotChickenUrl = 'https://www.clydeshotchicken.com/';
+  const portosBakeryUrl = 'https://www.portosbakery.com/';
+
+  // Variables starting with shabu will be referring to "House of Shabu Shabu II".
+  const shabuContentString = '<div id="shabu-content">' +
+      '<h1 class="firstHeading">House of Shabu Shabu II</h1>' +
+      '<div>' +
+      '<p><b>House of <a href="' + shabuShabuWikiUrl + '">Shabu Shabu</a> II</b> is a great place to ' +
+      'go to with a friend or a special someone, especially if you are into ' +
+      'the interactive experience of <a href="' + koreanBarbecueWikiUrl + '">Korean barbecue</a>. ' +
+      'The experience at House of Shabu Shabu II is similar in that ' +
+      'you choose from a selection of menus which include a multitude ' +
+      'of meats and vegetables and can order as much as you want! </p> ' +
+      '</div>' +
+      '</div>';
+
+  // Variables starting with boiling will be referring to "Boiling Point".
+  const boilingContentString = '<div id="boiling-content">' +
+      '<h1 class="firstHeading">Boiling Point</h1>' +
+      '<div>' +
+      '<p><b>Boiling Point</b> is an establishment that specializes in <a href="' + hotPotWikiUrl + '">Taiwanese hot soup</a> cuisine. ' +
```

Comment:
Consider using line breaks - I think this line is kind of long.

Reply:
Done.